css-mixins-1/Overview.bs

```diff
@@ -104,7 +104,7 @@ The ''@function'' rule defines a [=custom function=],
 and its syntax is:
 
 <pre class="prod def" nohighlight>
-&lt;@function> = @function <<function-name>> <<function-parameter-list>>? )
+&lt;@function> = @function <<function-name>>( <<function-parameter-list>>? )
```

Comment:
`<<function-name>>` is a `<function-token>`. It already includes a `(`. It would be nice if this was readily apparent, but I couldn't figure out something better than this.

Reply:
Ah sorry, that wasn’t clear to me. Took me some clicks to find it in css-values-3.

Will undo this change.